README.md

```diff
@@ -244,39 +244,44 @@ You can also add them to the end of `~/.profile` to set this by default in all s
 
 #### MacOS dependencies installation
 
-To install GCC and CMake on MacOS we recommend to use [Homebrew](https://brew.sh/) package manager. With Homebrew
+First of all, install Xcode Command Line Tools if you don't have them. Open terminal and run:
+```sh
+xcode-select --install
+```
+You'll see a panel that asks you to install Xcode Command Line Tools. Click 'Install' to begin the installation process and wait for it to complete.
```

Comment:
IMHO this feels like spoon-feeding.
I feel like
```md
#### Install Xcode Command Line Tools

Open Terminal and run:

```sh
xcode-select --install

Follow the prompts to install.
```

is sufficient